src/components/window-shade-button.tsx

```diff
@@ -0,0 +1,34 @@
+import * as React from "react";
+
+import * as css from "./window-shade.sass";
+
+import ContentConfigurations from "../config/content-configurations";
+
+interface IProps {
+  type: string;
+  onClick: () => void;
+}
+
+interface IState { }
+
+export default class WindowShade extends React.Component<IProps, IState> {
+
+  public render() {
+    const { onClick, type } = this.props;
+    const { icon, label, styleClassName } = ContentConfigurations[type];
+    const cssClassNames = [ css.windowShadeToggle, css[styleClassName] ];
+    const iconProps = {
+      width: "50px",
+      fill: "red"
+    };
```

Comment:
Note that we can use CSS for that instead (I've tried and it works). We generally tend to use CSS instead of inline styles, so that would be my preference too.

Reply:
That was the intent, here, too -- to move to it to CSS. At this point I just wanted to make sure props were being respected all they way into the SVG react object.